macOS/DuckDuckGo/Bookmarks/Services/LocalBookmarkStore.swift

```diff
@@ -738,8 +744,10 @@ final class LocalBookmarkStore: BookmarkStore {
             let processedErrors = CoreDataErrorsParser.parse(error: error)
 
             if AppVersion.runType.requiresEnvironment {
-                PixelKit.fire(DebugEvent(GeneralPixel.bookmarksSaveFailedOnImport, error: error),
-                           withAdditionalParameters: processedErrors.errorPixelParameters)
+                PixelKit.fire(
```

Comment:
For testing this specific instance you'll want to force an error in the import logic, but the change is the same as above so it should be fine..